projects/resettlement-passport-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/entity/CaseNote.kt

```diff
@@ -0,0 +1,98 @@
+package uk.gov.justice.digital.hmpps.entity
+
+import jakarta.persistence.*
+import org.hibernate.annotations.Immutable
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.LastModifiedBy
+import org.springframework.data.annotation.LastModifiedDate
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import org.springframework.data.jpa.repository.JpaRepository
+import uk.gov.justice.digital.hmpps.exception.NotFoundException
+import java.time.LocalDate
+import java.time.ZonedDateTime
+
+@EntityListeners(AuditingEntityListener::class)
+@Entity(name = "ContactEntity")
+@Table(name = "contact")
+class CaseNote(
+    @Id
+    @Column(name = "contact_id", updatable = false)
+    @SequenceGenerator(name = "contact_id_seq", sequenceName = "contact_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "contact_id_seq")
+    val id: Long = 0,
+
+    @ManyToOne
+    @JoinColumn(name = "offender_id")
+    val person: Person,
+
+    @ManyToOne
+    @JoinColumn(name = "contact_type_id", updatable = false)
+    val type: CaseNoteType,
+
+    @Lob
+    var notes: String,
+
+    @Column(name = "contact_date")
+    var date: LocalDate,
+
+    @Column(name = "contact_start_time")
+    var startTime: ZonedDateTime,
```

Comment:
Should these be `val`s?  There's a few more further down in the entity too.

Reply:
fixed